components/concept/awesomebar/src/main/java/mozilla/components/concept/awesomebar/AwesomeBar.kt

```diff
@@ -122,7 +122,10 @@ interface AwesomeBar {
         enum class Flag {
             BOOKMARK,
             OPEN_TAB,
-            CLIPBOARD
+            CLIPBOARD,
+            SyncedTabsDesktop,
+            SyncedTabsMobile,
+            SyncedTabsTablet
```

Comment:
```suggestion
            CLIPBOARD,
            SYNC_TAB
```

I don't think we need to separate each type here for the flag. Let's just use one and match the styling.